### ISSUE TITLE
Chatbot: Messages need an inline error state

### ISSUE THREAD

**Issue by kaylachumley** [patternfly/patternfly-design-kit#773]
Per request from the RHDH LS team, we need to create a message level inline error state for failed instances. 

example: when a user waits for the AI message streaming to complete but for some reason the streaming is stopped. In this case we may want to surface the error happened only on the last message from the bot/AI.

![Image](https://github.com/user-attachments/assets/cd507525-6290-4756-bd4b-8c4201e8f34d)
![Image](https://github.com/user-attachments/assets/e507f342-cd2d-41ce-afd3-7ab492aee7df)

**Comment by kaylachumley** [comment]
@andrew-ronaldson I opted to adopt the current pf text input error state for this. What are your thoughts? If none, this is ready to go for rebecca to implement. 

Dark:

![Image](https://github.com/user-attachments/assets/a99b7dfa-987d-40bf-9f70-3c5bba6307b8)

Light:

![Image](https://github.com/user-attachments/assets/8c1446bf-8134-47c5-b1cf-c81d0f4b0b38)

**Comment by kaylachumley** [comment]
Updated design per team feedback: 

Leans toward inline alert - no dismiss
- Reading left to right makes sense - icon position adds context
- No ability to dismiss
- Nicole wants a general inline alert that people can customize. Minimize tweaks out of the box. Keep Inline with chat

Using the inline alert component with no dismissal to start

![Image](https://github.com/user-attachments/assets/00728569-bf67-4974-be82-c67560e9a639)

**Comment by andrew-ronaldson** [comment]
Looks good to me. Is this going to be added as a "message type" variant in the Figma file?

**Comment by kaylachumley** [comment]
yes! @andrew-ronaldson  I added it in the message group as a bot error variant 

![Image](https://github.com/user-attachments/assets/c960ad34-6adc-4dfc-920c-1ddbf5a49166)

**Comment by andrew-ronaldson** [comment]
Cool beans. Closing as donezo!
